subprojects/shipkit/src/main/groovy/org/shipkit/gradle/configuration/ShipkitConfiguration.java

```diff
@@ -31,7 +32,11 @@
     private String previousReleaseVersion;
 
     public ShipkitConfiguration() {
-        this(new ShipkitConfigurationStore());
+        this(new EnvVariables());
+    }
+
+    public ShipkitConfiguration(EnvVariables envVariables) {
```

Comment:
Using internal EnvVariable in public api is not good. I will rework this one later on today.

Reply:
What if you make this constructor package-private and create ShipkitConfigurationFactory in the same package, that would be located in test code, not production one? This way it can be used across all tests and it is not a part of public API.